src/config.ts

```diff
@@ -35,11 +35,7 @@ let config: InternalConfig = {
     const error = new Error(
       [
         message,
-        prettifiedDOM.length > 0
-          ? `Ignored nodes: comments, <script />, <style />\n${prettyDOM(
-              container,
-            )}`
-          : null,
+        `Ignored nodes: comments, <script />, <style />\n${prettifiedDOM}`,
```

Comment:
Removed the handling of `prettifiedDOM.length === 0` from the default `getElementError` function. From what I could tell, the only case where that was true was when the function was called from within Cypress. Since we've removed the Cypress special-casing in this PR, this branch no longer seemed useful.